### PR TITLE
Sync installer

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by howardjohn** [comment]
blocking https://github.com/istio/istio/issues/19746

File: pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -562,7 +562,14 @@ message GlobalConfig {
   // Specifies the Istio control plane’s telemetry Pod IP address or remote cluster DNS resolvable hostname
   string remoteTelemetryAddress = 49;
 
-  // The next available key is 54
+  IstiodConfig istiod = 54;
```
Comment (elfinhe) [inline pkg/apis/istio/v1alpha1/values_types.proto]: can we also add comments here?

Comment (howardjohn) [reply pkg/apis/istio/v1alpha1/values_types.proto]: done

**Review by elfinhe — APPROVED** [review]
LGTM. However, there is some relation between the enablements of istiod and pilot/galley (e.g., pilot is set to disabled but istiod is set to enabled). we need add more logic or docs to handle that.

**Comment by howardjohn** [comment]
Lets go with the minimal approach for now and make it "smarter" later.

**Review by richardwxn — APPROVED** [review]

**Pull request merged**